doc/example-config-files/jitsi.example.com.example

```diff
@@ -6,28 +6,18 @@ server {
     root /srv/jitsi.example.com;
     index index.html;
 
-    location ~ ^/([a-zA-Z0-9=\?]+)$ {
+    location ~ ^/([a-zA-Z0-9_-]+)$ {
         rewrite ^/(.*)$ / break;
-        }
+    }
 
     location / {
         ssi on;
     }
 
     # BOSH
-    location /http-bind {
+    location = /http-bind {
         proxy_pass      http://localhost:5280/http-bind;
         proxy_set_header X-Forwarded-For $remote_addr;
         proxy_set_header Host $http_host;
     }
-
-    # xmpp websockets
-    location /xmpp-websocket {
-        proxy_pass http://localhost:5280;
-        proxy_http_version 1.1;
-        proxy_set_header Upgrade $http_upgrade;
-        proxy_set_header Connection "upgrade";
-        proxy_set_header Host $host;
-        tcp_nodelay on;
-    }
 }
```

Comment:
We do not want to remove the websocket part, for example on meet deployment it is used by the iOS client.